src/app/_components/TextLink/TextLink.module.scss

```diff
@@ -0,0 +1,3 @@
+.base {
+  @apply text-brand-300 focus:brand-outline hover:underline;
+}
```

Comment:
Love the idea of moving the style into separate CSS files!

I'm not sure we need CSS modules here, though.

Since we use `@apply`, we get the Tailwind optimization out of the box, and we don't need to import the CSS in components that need it.

Reply:
I'm not entirely sure what the suggestion is, to move them to a regular CSS file?

If we do that, we lose the scoping benefits of CSS Modules. I also think that React/Next.js best practices usually favor scoped styling solutions.

Add them to `tailwind.config.js`?

We can talk about this a bit more, but moving forward as I don't think this is a blocker!